tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/ColorPacket.cs

```diff
@@ -0,0 +1,57 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+using static System.Runtime.Intrinsics.X86.Avx;
+using System.Runtime.Intrinsics.X86;
+using System.Runtime.Intrinsics;
+using System.Runtime.CompilerServices;
+
+using ColorPacket256 = VectorPacket256;
+
+internal static class ColorPacket256Helper
+{
+    public static Int32RGBPacket256 ConvertToIntRGB(this VectorPacket256 colors)
+    {
+        var one = SetAllVector256<float>(1.0f);
+        var max = SetAllVector256<float>(255.0f);
```

Comment:
Is a `static readonly` more efficient, since we don't have JIT support for this yet?

Reply:
and, even if we did have JIT support, since the constants are per method, a static readonly might still be better...

Reply:
Thanks, will cache them in static fields.